modules/fts/pages/fts-monitor.adoc

```diff
@@ -0,0 +1,144 @@
+= Monitor 
+
+== Monitor runtime queries
+
+Users of the search service might want to monitor the currently running active queries on the Full Text Search (FTS) search page. Monitoring helps get insights into slow queries or for other debugging purposes.
```

Comment:
Trailing "search page" is redundant here. 

Users of the search service might want to monitor the currently running active queries on the Full Text Search (FTS) service. looks enough?

Reply:
Proposal: Users of the full text search service might want to monitor the currently running active queries. Monitoring helps get insights into slow queries or for other debugging purposes.

Reply:
```suggestion
Users of the full text search service might want to monitor the currently running active queries.
Monitoring helps get insights into slow queries, or for other debugging purposes.
```